src/CaptureClient/CompositeEventProcessorTest.cpp

```diff
@@ -15,16 +15,28 @@ class MockEventProcessor : public CaptureEventProcessor {
 };
 
 TEST(CompositeEventProcessor, Smoke) {
-  MockEventProcessor processor1;
-  MockEventProcessor processor2;
-  MockEventProcessor processor3;
+  // CompositeEventProcessor assumes ownership of event processors used to construct it.
```

Comment:
```suggestion
  // CompositeEventProcessor takes ownership of event processors used to construct it.
```
"assume" is kind of ambiguous.